src/databricks/azext_databricks/_validators.py

```diff
@@ -0,0 +1,54 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import random
+import string
+
+
+def id_generator(size=13, chars=string.ascii_lowercase + string.digits):

```

Comment:
Just curious, why 13? Most sizes are power of 2

Reply:
This is following the default behavior in Azure Portal.